packages/cardano-services/src/ChainHistory/DbSyncChainHistory/util.ts

```diff
@@ -86,8 +86,8 @@ export const applyPagination = (ids: Cardano.TransactionId[], { startAt, limit }
  * certificates_.
  *
  * Example: given an array of `reg_cert` and an array of `vote_deleg_cert`, the result is an array of:
- * - all input `reg_cert` which can't be neither `vote_reg_deleg_cert` nor `stake_vote_reg_deleg_cert,
- * - all input `vote_deleg_cert` which can't be neither `vote_reg_deleg_cert` nor `stake_vote_reg_deleg_cert,
+ * - all input `reg_cert` which can't be neither `vote_reg_deleg_cert` nor `stake_vote_reg_deleg_cert`,
```

Comment:
Lots of negation in this comment :sweat_smile: Is this what it means?
```suggestion
 * - all input `reg_cert` which can be either `vote_reg_deleg_cert` or `stake_vote_reg_deleg_cert`,
```

Reply:
e99b89e3f8a should solve this.

Reply:
Sorry but it's still confusing. Does it mean:
- can be either A or B
- can be **n**either A or B (which is how I currently read it with "can **'t** be")

Reply:
8f1ca36b86b should solve this